### PR TITLE
react-redux-login

### PR DESCRIPTION
**Pull request by Vahe101**

### CODE REVIEW

File: src/Router.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import { Home } from "./components/Home/index.js";
+import { Header } from "./components/common/Header";
+import { Login } from "./components/Login";
+import { Products } from "./components/Product";
+import { Workers } from "./components/Worker";
+import { check } from "./redux/auth/authTypes";
+import allActions from "./redux/auth/actions/index";
+
+export const Routers = () => {
+  const isAuth = useSelector((state) => state.currentUser);
```
Comment (edotnet) [inline src/Router.js]: rename `currentUser` module to `auth`

File: src/Router.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  const dispatch = useDispatch();
+
+  const action = {
```
Comment (edotnet) [inline src/Router.js]: declare actions inside redux folder

File: src/components/About/Header/index.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import "../index.scss";
```
Comment (edotnet) [inline src/components/About/Header/index.js]: Have components css file inside same folder

File: src/components/Login/index.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+  const history = useHistory();
+  const dispatch = useDispatch();
+
+  const login = async () => {
+    const user = {
+      username: email,
+      password: password,
+    };
+
+    const action = {
```
Comment (edotnet) [inline src/components/Login/index.js]: declare inside redux folder

File: src/components/Product/Content/index.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const Content = (props) => {
```
Comment (edotnet) [inline src/components/Product/Content/index.js]: remove `props` param since its not used

File: src/components/Worker/Content/index.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+          <li>2. Edik Baxdasaryan - CTO at Volatile Solutions</li>
+          <li>3. Vahe Davtyan - Software Developer</li>
+          <li>4. Karen Avdalian - Software Developer</li>
+          <li>5. Arayik Hovhannisyan - Software Developer</li>
+          <li>6. Gurgen Baxdasaryan - Software Developer</li>
+          <li>7. Hayk Mkrtchyan - Software Developer</li>
+          <li>8. Hayk Qaryan - Software Developer</li>
+          <li>9. Sergey Ghazaryan - Software Developer</li>
+          <li>10. Suren Melqonyan - Software Developer</li>
+          <li>11. Arpi Tonoyan - Part Time Developer</li>
```
Comment (edotnet) [inline src/components/Worker/Content/index.js:20]: update, now we have new teammate

File: src/redux/auth/reducers/currentUser.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  type: "",
+  user: {},
+  loggedIn: false,
+};
+
+export const currentUser = (state = initialState, action) => {
+  switch (action.type) {
+    case log_in:
+      return {
+        type: action.type,
```
Comment (edotnet) [inline src/redux/auth/reducers/currentUser.js]: you dont need to store action type

**Review by edotnet — CHANGES_REQUESTED** [review]

File: src/Router.js (hunk 1 above)
Comment (Vahe101) [reply src/Router.js]: done

File: src/components/About/Header/index.js (hunk 3 above)
Comment (Vahe101) [reply src/components/About/Header/index.js]: accidentally put the file in the wrong place, my fault, the rest did so

File: src/components/Worker/Content/index.js (hunk 6 above)
Comment (Vahe101) [reply src/components/Worker/Content/index.js:20]: mmm

File: src/redux/auth/reducers/currentUser.js [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+        type: log_in,
+        user: action.payload,
+        loggedIn: true,
+      };
+    default:
+      return state;
+  }
+};
+
+export const inspector = (state = initialState, action) => {
```
Comment (edotnet) [inline src/redux/auth/reducers/currentUser.js]: You dont need this one, remove

File: src/redux/auth/reducers/currentUser.js [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+  type: "",
+  user: {},
+  loggedIn: false,
+};
+
+export const auth = (state = initialState, action) => {
+  switch (action.type) {
+    case log_in:
+      return {
+        type: log_in,
```
Comment (edotnet) [inline src/redux/auth/reducers/currentUser.js]: dont store type in state

**Review by edotnet — CHANGES_REQUESTED** [review]

**Review by edotnet — APPROVED** [review]

**Pull request merged**